README.md

```diff
@@ -153,6 +161,14 @@ export SQLA_CONN=<psql:address-to-box>
    
    *Note: 18F team members should not set these environment variables. 18F and FEC team members will also have additional environment variables to set up. Please reach out to a team member for detailed information.*
 
+4. If you are using database replicas/mirrors you can also restrict connections to them to be asynchronous tasks only by running:
+
+   ```
+   export SQLA_RESTRICT_FOLLOWER_TRAFFIC_TO_TASKS=True
```

Comment:
Yes, you're right, and I'm just not quite sure how best to approach this given that an environment variable is just a string that needs to be cast if it's being used in any other capacity (and probably over thinking it).  What do you recommend setting this to instead?  Would using just a descriptive word such as `on` or `enabled` make more sense here?